### PR TITLE
ptyprocess: bump to 0.7.0 and drop -27

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/python/ptyprocess/history [hunk 1]
```diff
@@ -1 +1,2 @@
+library/python/ptyprocess-27@0.6.0-2020.0.1.4
 library/python/ptyprocess-34@0.6.0,5.11-2020.0.1.1
```
Comment (AndWac) [inline components/python/ptyprocess/history:2]: ,5.11 is missing here (the 2nd has it)

Comment (alarcher) [reply components/python/ptyprocess/history:2]: I think we concluded that short fmris are supported since pkg was modified?

**Comment by AndWac** [comment]
Am 8/6/21 um 11:35 PM schrieb Aurélien Larcher:
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In components/python/ptyprocess/history
> <https://github.com/OpenIndiana/oi-userland/pull/6967#discussion_r684515409>:
>
> > @@ -1 +1,2 @@
> ***@***.***
>   ***@***.***,5.11-2020.0.1.1
>
> I think we concluded that short fmris are supported since pkg was
> modified?
>
Yes, I think they work. The last thing I remember was that alp wanted
the long form. Maybe I misunderstood him, though.

**Review by AndWac — APPROVED** [review]

**Comment by alarcher** [comment]
> Yes, I think they work. The last thing I remember was that alp wanted the long form. Maybe I misunderstood him, though.

Ah ok :) I actually started using the short form since he used it ;) I'll try to find the conversation.

**Pull request merged**